cloud-controller-manager/do/firewall_controller.go

```diff
@@ -329,12 +330,22 @@ func (fm *firewallManager) createReconciledFirewallRequest(serviceList []*v1.Ser
 			}
 			if lbType == godo.LoadBalancerTypeRegionalNetwork && lbNetwork == godo.LoadBalancerNetworkTypeExternal {
 				// Add the health check port
-				_, healthCheckPort := healthCheckPathAndPort(svc)
-				if healthCheckPort == 0 {
+				_, hcp := healthCheckPathAndPort(svc)
+				if hcp == 0 {
 					continue
 				}
-				loadBalancerPorts[portProtocol{protocol: "tcp", port: healthCheckPort}] = struct{}{}
-
+				pp := portProtocol{
+					protocol: "tcp",
+					port:     hcp,
+				}
+				if hcp == kubeProxyHealthPort {
```

Comment:
I'm sorry for missing some context, if the health check port is something other than kube-proxy, why are we okay allowing all ipv4 and ipv6 for other health check ports defined?